<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>618</width>
    <height>452</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>618</width>
    <height>452</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ramki v0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Obrazek</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wysokość</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Szerokość</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editObrazWys">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editObrazSzer">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Paspartu</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Szerokość</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPaspartu">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Ramka</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Szerokość</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editRamkaSzer">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cena</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editRamkaCena">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>zł</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Cena</string>
    </property>
    <widget class="QLineEdit" name="textCena">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Szkło</string>
    </property>
    <widget class="QRadioButton" name="radioSzkloBrak">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Brak</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioSzkloFloat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Szkło float</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioSzkloAnty">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Szkło antyreflex</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkKlejenie">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Klejenie</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPodporka">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Podpórka</string>
    </property>
   </widget>
   <widget class="podglad" name="wizualizacja" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>390</width>
      <height>390</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOprogramie"/>
    <addaction name="separator"/>
    <addaction name="actionZako"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionOprogramie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionZako">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>podglad</class>
   <extends>QWidget</extends>
   <header>podglad.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>editObrazWys</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editObrazSzer</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editPaspartu</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editRamkaSzer</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkPodporka</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkKlejenie</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editRamkaCena</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSzkloBrak</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSzkloFloat</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSzkloAnty</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>obliczCene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>obliczCene()</slot>
 </slots>
</ui>
